Book-A-Desk.Api/EmailNotification.fs

```diff
@@ -0,0 +1,46 @@
+namespace Book_A_Desk.Api
+
+open System
+open System.Net.Mail
+open Book_A_Desk.Api.Models
+open Book_A_Desk.Domain.QueriesHandler
+open Book_A_Desk.Domain.Office.Domain
+
+type EmailNotification =
+    {
+        SendEmailNotification: Booking -> unit
+    }
+
+module EmailNotification =
+    let initialize getEmailServiceConfiguration getOffices =         
```

Comment:
Small detail: We usually call the init of these services "provide" :)

Reply:
FYI: In the Broadsign/Ayuda codebase we use both, sometime 'provide' and sometimes 'initialize'. But we can try to find a common sense in this repo.

Reply:
Interesting! I don't really mind either, but we should really stick to one for those that belong together

Reply:
To Do: Rename to "provide".